Makefile

```diff
@@ -57,6 +57,16 @@ update:
 start: update
 	CONTIV_NODE_OS=${CONTIV_NODE_OS} vagrant up
 
+#kubernetes demo targets
+k8s-cluster:
```

Comment:
Can we make these targets consistent with docker stack, which has following meanings to them:
- 'make start' = bring up vms, start basic services (does not bring up the netplugin, used by developers typically
- 'make stop' = stop and destroy vagrant vms
- 'make demo' = brings entire demo setup with services running, etc.

It seems like in k8s case:
- 'make kds-cluster' is equivalent to 'make demo'
- 'make k8s-destroy' is equivalent to 'make stop'
- 'make demo' and 'make demo-start' does not have an equivalent one which copies the appropriate docker images, etc. to get the demo going on. I am not very particular if these are not kept in contiv/demo repo; but name conflict is something to be avoided. 
@mapuri, @shaleman, @erikh  - any comments?